src/Microsoft.AspNetCore.Razor.Design/build/netstandard2.0/Microsoft.AspNetCore.Razor.Design.Compilation.targets

```diff
@@ -47,7 +47,6 @@
           Outputs="@(RazorIntermediateAssembly);
                    @(_RazorDebugSymbolsIntermediatePath);
                    $(NonExistentFile)"
-          DependsOnTargets="RazorCoreGenerate"
```

Comment:
I did a little general cleanup here. We don't really plan for a target like `RazorCoreCompile` to be directly invoked, so I'm not trying to make it look like it's supported.